### PR TITLE
feat: Simplify usage

### PR DESCRIPTION
**Pull request by KostyaCholak**

### CODE REVIEW

File: src/buttons.py [hunk 1]
```diff
@@ -17,40 +17,13 @@
...
-              default_daily_target='60 or 2'),
-        Track(name="Content Creation",
-              questions_needed=['description', 'metric', 'target', 'frequency'],
-              default_tracking_metric="e.g. posts, videos, blogs", 
-              default_daily_target='1 or 2'),
-        Track(name="Other",
-              questions_needed=['description', 'metric', 'target', 'frequency'],
-              default_tracking_metric="Smiles", 
-              default_daily_target=10),
+        Track(name="Setup"),
```
Comment (thebadcoder96) [inline src/buttons.py:26]: I am not sure but I still think we need to ask all the questions in the form for setup.

Comment (KostyaCholak) [reply src/buttons.py:26]: The idea is to simplify the setup in order to minimise the number of places where people can drop off. Then we will collect monitoring where people get lost. Right now we have a really low engagement with the bot

Comment (thebadcoder96) [reply src/buttons.py:26]: I understand. We have been receiving some messages about users having trouble setting goals and completing profiles which may be one of the reasons!

**Review by thebadcoder96 — APPROVED** [review]

**Comment by thebadcoder96** [comment]
there are some conflicts in this branch. I had looked through the initial commits and approved them.

**Pull request merged**